src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF-TRAINING.json

```diff
@@ -0,0 +1,95 @@
+{
+    "Description": "MLPerf Training GPU Performance Workload",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "10:00:00",
+        "SupportedPlatforms": "linux-x64",
+        "SupportedOperatingSystems": "Ubuntu",
+        "SpecialRequirements": "This is an Nvidia GPU-specialized workload. It depends upon the system having an Nvidia GPU card/chip. Requires an OSDisk of >=256GB size and Data disk of >=8TB size"
+    },
+    "Parameters": {
+        "Username": "",
+        "DiskFilter": "SizeGreaterThan:1000gb",
+        "LinuxCudaVersion": "12.0",
+        "LinuxDriverVersion": "525",
+        "LinuxLocalRunFile": "https://developer.download.nvidia.com/compute/cuda/12.0.0/local_installers/cuda_12.0.0_525.60.13_linux.run"
+    },
+    "Actions": [
+        {
+            "Type": "MLPerfTrainingExecutor",
+            "Parameters": {
+                "Scenario": "training-mlperf-bert-batchsize-45-gpu-8",
+                "Model": "bert",
+                "Username": "$.Parameters.Username",
+                "DiskFilter": "$.Parameters.DiskFilter",
+                "BatchSize": "45",
+                "Implementation": "pytorch-22.09",
+                "ContainerName": "language_model",
+                "DataPath": "mlperf-training-data-bert.1.0.0",
+                "GPUNum": "8",
+                "ConfigFile": "config_DGXA100_1x8x56x1.sh"
```

Comment:
Great! Do we have other config files we could use? Other than A100?

Reply:
We do have config files for h100 preview too. The config files also change with the number of GPUs and batch size used for training. The config files follow the naming convention, config_<GPU>_<number of nodes>_<number of GPUs>_<batch size>_<gradient acc steps>.sh